tests/test_util.py

```diff
@@ -0,0 +1,110 @@
+from conftest import _make_nb_obj
+
+import pytest
+from ploomber_engine._util import parametrize_notebook, add_debuglater_cells
+
+
+@pytest.mark.parametrize(
+    "nb, params_idx, cells_total",
+    [
+        [_make_nb_obj(["1 + 1"]), 0, 2],
+        [_make_nb_obj([("code", "1 + 1", {"tags": ["parameters"]})]), 1, 2],
+        [_make_nb_obj(["21 + 21", ("code", "1 + 1", {"tags": ["parameters"]})]), 2, 3],
+        [_make_nb_obj(["# parameters\n1 + 2"]), 1, 2],
+        [_make_nb_obj(["1 + 1", "# parameters\n1 + 2"]), 2, 3],
+        [_make_nb_obj(["#PARAMETERS\n1 + 2"]), 1, 2],
+        [_make_nb_obj(["1 + 1", "#PARAMETERS\n1 + 2"]), 2, 3],
```

Comment:
Since users might use `# parameters`, I think we can add a test for lowercase as well.

Reply:
there is one lowercase test in line 14, do you think we need more?

Reply:
didn't notice it. definitely enough.